### ISSUE TITLE
`a7e99bf` broke the template

### ISSUE THREAD

**Issue by vbrandl** [RatanShreshtha/DeepThought#18]
The commit a7e99bf188b88d4fcb4cc0ae4d8bf9c84df29e61 broke the theme, especially the linked change: https://github.com/RatanShreshtha/DeepThought/commit/a7e99bf188b88d4fcb4cc0ae4d8bf9c84df29e61#diff-690b81fad8df2a1f1ce37c846641d9247e3472d244f17039d2910a5ed0c98d5eR4

Removing `{% set lang = lang | default(value=config.default_language) %}` from `base.html` causes the following error:

```
Error: Failed to render '404.html' (error happened in 'base.html').
Reason: Variable `lang` not found in context while rendering '404.html'
```

Putting this line back into the template solves the problem.

**Comment by RatanShreshtha** [comment]
Latest version of zola has fix for this

**Comment by omofolarin** [comment]
Fix hasn't been applied

**Comment by ian-fox** [comment]
@omofolarin fix seems to be in zola 0.13.x, if you're using dnf or something else that still has 0.12.x try building from source or downloading a release binary directly from github.
